README.md

```diff
@@ -23,6 +23,10 @@ A lightweight page object implementation with a focus on simplicity and extensib
     - [Lightweight (no jQuery)](#lightweight-no-jquery)
 - [Why `fractal`?](#why-fractal)
 
+## Example
```

Comment:
Cool, looks good! A couple of final nits:

* Can we call the section `Examples` instead of `Example` since there are multiple?
* The table of contents is auto-generated (I use https://github.com/yzhang-gh/vscode-markdown in VSCode to generate/keep in sync). So any section heading should either be in the table of contents or have a `<!-- omit in toc -->` so it will be omitted when auto-generating. I kinda think it makes sense to include this in the table of contents -- what do you think?

Reply:
Agree and agree. Did not realize that the TOC need to be updated. Thanks, learned something new :)